### ISSUE TITLE
Crash playing on Rank

### ISSUE THREAD

**Issue by kjsunday** [TekkenOverlay/TekkenOverlay#137]
Issue still hasn't fixed when playing rank. After loading screen in match it crashes. I only used Next match information, 165hz, and interpolation.

**Comment by xNadeemx** [comment]
I'm currently using [TekkenOverlay-5.11.9-[2023-10-21] with T7 and it works fine for me playing ranked, no crashes. 175hz here with interpolation. I wonder if something else is causing your game to crash with the overlay.

**Comment by kjsunday** [comment]
Yeah, Idk what's the problem. Do you know where's the crashlog located?
